runtime/libpgmath/lib/common/mthdecls.h

```diff
@@ -34,7 +34,7 @@
 #define I_SIZE 2 /* i[] has 2 elements and 128 bits */
```

Comment:
shouldn't those new global declarations be guarded by `#ifdef`'s across this file?

Reply:
I have rewritten this part of the code.